python/lsst/pipe/tasks/skyCorrection.py

```diff
@@ -344,6 +384,14 @@ def run(self, calExps, calBkgs, skyFrames, camera):
             skyCorrExtras, camera, self.config.binning, ids=calExpIds, refExps=calExps
         )
 
+        # Question: should this uncorrection be done before the binning and display?
```

Comment:
I'd guess we probably want the display images to be flattened for background-subtraction.